packages/project-starter/e2e/starter-plugin.test.ts

```
@@ -1,15 +1,49 @@
+// Define a minimal TestSuite interface that matches what's needed
+interface TestSuite {
+  name: string;
+  description: string;
+  tests: Array<{
+    name: string;
+    fn: (runtime: any) => Promise<any>;
+  }>;
+}
+
+// Define minimal interfaces for the types we need
+interface UUID extends String {}
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)